sc2repo/build.gradle

```diff
@@ -33,15 +33,15 @@ dependencies {
     
     compile 'org.opencadc:cadc-util:1.+'
     runtime 'org.opencadc:cadc-log:[1.0.1,)'
-    compile 'org.opencadc:caom2-repo-server:[2.3.7,3.0)'
+    compile 'org.opencadc:caom2-repo-server:[2.3.8,3.0)'
     compile 'org.opencadc:cadc-access-control:[1.1.2,)'
     compile 'org.opencadc:cadc-access-control-identity:1.+'
     compile 'org.opencadc:cadc-vosi:[1.0.1,2.0)'
     
     testCompile 'junit:junit:4.+'
     
     intTestCompile 'org.opencadc:cadc-test-vosi:[1.0.1,)'
-    intTestCompile 'org.opencadc:caom2-test-repo:[1.0.15,)'
+    intTestCompile 'org.opencadc:caom2-test-repo:[1.0.17,)'
 }
 
```

Comment:
do you want to update the war file version?

Reply:
we aren't really using that  - it gets lost at build - so I haven't been bothering